src/exercises/direct/consumer.ts

```diff
@@ -0,0 +1,39 @@
+import { connect } from "amqplib";
+import { random } from "lodash";
+import yargs from "yargs";
+
+random();
+
+export function wait(timeout: number) {
+  return new Promise((res, _) => {
+    return setTimeout(res, timeout);
+  });
+}
+
+async function main() {
+  //Using yargs. To set the the queue pass as argument --queue <queue_number>
+  //<queue_number>
+  const argv = yargs(process.argv.slice(2))
+    .options({
+      queue: { type: "string", default: "1" },
```

Comment:
```suggestion
      queue: { type: "string", default: "q1" },
```
I suggest to change the default in q1